### PR TITLE
Fix list and typo in article gga-v1

### PR DESCRIPTION
**Pull request by zckrs**
List render is wrong. I fix problem with copy format of article `2014-05-12-generator-karma-rewrite.md`

### CODE REVIEW

**Comment by zckrs** [comment]
ping @eddiemonge

File: app/_posts/blog/2015-10-15-generator-gulp-angular-v1.md [hunk 1]
```diff
@@ -15,35 +15,37 @@ At first, we wanted to make a good merge of [generator-gulp-webapp](https://gith
...
+ * Use automatic injection in different parts of the project: script tags both vendor and sources in the index.html, styles files, vendor, css or preprocessed.
+ * Provide test coverage, as perfect as possible, for both the generator and also code it generated.
 
 ### Technologies supported
 
-We are not joking around when we talk about this being a stable version. We integrated lots of technologies and languages, from CoffeeSript to TypeScript, to Sass to Stylus. The number of combinations exceeds several million! We wrote tests, documentation and fixed issues for 12 minor versions and 2 release candidates, to be able to deliver a perfectly configured seed project, no matter the options you choose.
+We are not joking around when we talk about this being a stable version. We integrated lots of technologies and languages, from CoffeeScript to TypeScript, to Sass to Stylus. The number of combinations exceeds several million! We wrote tests, documentation and fixed issues for 12 minor versions and 2 release candidates, to be able to deliver a perfectly configured seed project, no matter the options you choose.
```
Comment (RaphaelYan) [inline app/_posts/blog/2015-10-15-generator-gulp-angular-v1.md]: `2 release` => `2 releases`

Comment (zckrs) [inline app/_posts/blog/2015-10-15-generator-gulp-angular-v1.md]: I'm not sure about that.

**Comment by SBoudrias** [comment]
Thanks

**Pull request merged**